browser/src/index.tsx

```diff
@@ -251,6 +251,10 @@ const start = (args: string[]) => {
             return
         }
 
+        if (key === "|") {
+            pluginManager.findAllReferences()
+        }
```

Comment:
Can I ask why you picked `|` as the key to invoke this feature?  Is this a common key to invoke find-all-references?  I'm surprised you'd map this key directly given that `|` is already mapped by default in NeoVim.